fragments/app-sso-client/LICENSE

```diff
@@ -0,0 +1,16 @@
+MIT No Attribution
```

Comment:
I *think* it may be OK to move the LICENSE up at the root of the repo, which would have the benefit of not requiring to deal with it in the accelerator.yaml files.  IANAL though

Reply:
I think we want the license file to be part of the generated project, but IANAL either.

Reply:
Wouldn't the LICENSE file come from the host accelerator rather than (multiple) fragments, though?